glide.yaml

```diff
@@ -50,9 +50,9 @@ import:
   version: a8983c062be4b565d723c478922d7736e04fdba4
 - package: github.com/Masterminds/semver
   version: v1.3.1
+- package: github.com/imdario/mergo
+  version: ^0.2.2
 testImport:
 - package: github.com/onsi/gomega
 - package: github.com/onsi/ginkgo
-  version: v1.3.1
```

Comment:
then you are running into the exact problem i was seeing earlier when i was trying to get glide to reconcile